exercises/simple-cipher/src/test/java/IncorrectKeyCipherTest.java

```diff
@@ -0,0 +1,54 @@
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Step 3: Incorrect key cipher
+ */
```

Comment:
Having the step number in the test file names might make it easier for users to understand what each file is for? And also the hints.md file refers to those file names 🙂